impl/maven-impl/src/main/java/org/apache/maven/impl/resolver/scopes/Maven3ScopeManagerConfiguration.java

```diff
@@ -89,33 +84,37 @@ public BuildScopeSource getBuildScopeSource() {
 
     @Override
-    public Collection<DependencyScope> buildDependencyScopes(InternalScopeManager internalScopeManager) {
-        ArrayList<DependencyScope> result = new ArrayList<>();
-        result.add(internalScopeManager.createDependencyScope(DS_COMPILE, true, all()));
+    public Collection<org.eclipse.aether.scope.DependencyScope> buildDependencyScopes(
+            InternalScopeManager internalScopeManager) {
+        ArrayList<org.eclipse.aether.scope.DependencyScope> result = new ArrayList<>();
+        result.add(internalScopeManager.createDependencyScope(DependencyScope.COMPILE.id(), true, all()));
         result.add(internalScopeManager.createDependencyScope(
-                DS_RUNTIME, true, byBuildPath(CommonBuilds.BUILD_PATH_RUNTIME)));
+                DependencyScope.RUNTIME.id(), true, byBuildPath(CommonBuilds.BUILD_PATH_RUNTIME)));
         result.add(internalScopeManager.createDependencyScope(
-                DS_PROVIDED,
+                DependencyScope.PROVIDED.id(),
                 false,
                 union(
                         byBuildPath(CommonBuilds.BUILD_PATH_COMPILE),
                         select(CommonBuilds.PROJECT_PATH_TEST, CommonBuilds.BUILD_PATH_RUNTIME))));
         result.add(internalScopeManager.createDependencyScope(
-                DS_TEST, false, byProjectPath(CommonBuilds.PROJECT_PATH_TEST)));
+                DependencyScope.TEST.id(), false, byProjectPath(CommonBuilds.PROJECT_PATH_TEST)));
         result.add(internalScopeManager.createSystemDependencyScope(
-                DS_SYSTEM, false, all(), ArtifactProperties.LOCAL_PATH));
+                DependencyScope.SYSTEM.id(), false, all(), MavenArtifactProperties.LOCAL_PATH));
         return result;
     }
 
     @Override
-    public Collection<ResolutionScope> buildResolutionScopes(InternalScopeManager internalScopeManager) {
-        Collection<DependencyScope> allDependencyScopes = internalScopeManager.getDependencyScopeUniverse();
-        Collection<DependencyScope> nonTransitiveDependencyScopes =
+    public Collection<org.eclipse.aether.scope.ResolutionScope> buildResolutionScopes(
+            InternalScopeManager internalScopeManager) {
+        Collection<org.eclipse.aether.scope.DependencyScope> allDependencyScopes =
+                internalScopeManager.getDependencyScopeUniverse();
+        Collection<org.eclipse.aether.scope.DependencyScope> nonTransitiveDependencyScopes =
                 allDependencyScopes.stream().filter(s -> !s.isTransitive()).collect(Collectors.toSet());
-        DependencyScope system =
-                internalScopeManager.getDependencyScope(DS_SYSTEM).orElse(null);
+        org.eclipse.aether.scope.DependencyScope system = internalScopeManager
+                .getDependencyScope(DependencyScope.SYSTEM.id())
+                .orElse(null);
 
-        ArrayList<ResolutionScope> result = new ArrayList<>();
+        ArrayList<org.eclipse.aether.scope.ResolutionScope> result = new ArrayList<>();
```

Comment:
```suggestion
        ArrayList<ResolutionScope> result = new ArrayList<>();
```
might use import again.